json-java.gemspec

```diff
@@ -10,7 +10,6 @@ spec = Gem::Specification.new do |s|
   s.email = "dev+ruby@mernen.com"
   s.homepage = "http://json-jruby.rubyforge.org/"
   s.platform = 'java'
-  s.rubyforge_project = "json-jruby"
```

Comment:
This change as already been made: https://github.com/flori/json/commit/7e1e97dc11c5aaff76580310b7402ca13f5e62e2